condensedinlinepanel/static/condensedinlinepanel/src/components/Card.tsx

```diff
@@ -105,7 +105,7 @@ export class Card extends React.Component<CardProps, CardState> {
         for (let fieldName in this.props.form.fields) {
             let fieldElement = document.getElementById(`${this.props.formPrefix}-${fieldName}`);
 
-            if (fieldElement instanceof HTMLInputElement) {
+            if (fieldElement instanceof HTMLInputElement || fieldElement instanceof HTMLTextAreaElement) {
```

Comment:
Looks like I forgot to add ``|| fieldElement instanceof HTMLSelectElement`` here